bitnami/kafka/values.yaml

```
@@ -116,6 +116,11 @@ heapOpts: -Xmx1024m -Xms1024m
 ##
 deleteTopicEnable: false
 
+## Switch to enable auto creation of topics.
+## Enabling auto creation of topics not recommended for production or similar environments.
+##
+autoCreateTopicsEnable: false
```

```suggestion
autoCreateTopicsEnable: true
```

I agree not enabling the auto-creation of topics for production environment but yes for testing purposes. In that case, I would enable that feature in the `values.yaml` and disable it in the `values-production.yaml`.

In that case, can you also add the difference in this [README section](https://github.com/bitnami/charts/tree/master/bitnami/kafka#production-configuration-and-horizontal-scaling)?

---

I have edited `values.yaml` as you suggested, and updated [README section](https://github.com/bitnami/charts/tree/master/bitnami/kafka#production-configuration-and-horizontal-scaling) too